### DISCUSSION TITLE
Beacon or connection

### DISCUSSION THREAD

**Discussion by sprijk** [KrystianD/bm2-battery-monitor#21] [General]
Hi @KrystianD!

I was wondering if this bm2 battery monitor is also advertising the voltage as a beacon, or if it only exposes battery voltage once connected and BLE notify is activated. 

If it is doing advertising of the voltage as well, is it using the same encryption or how from what bytes would I extract the voltage information? 

Thanks in advance and best regards, 

Simon

**Comment by KrystianD** [comment]
Hi, as far as I remember, it doesn't advertise it as a becaon.

**Reply by sprijk** [reply]
got it! thanks
